src/autoComplete/AutoCompleteViewWrapper.js

```diff
@@ -4,7 +4,7 @@ import { StyleSheet, View } from 'react-native';
 
 import type { InputSelectionType, Narrow } from '../types';
 import AutoCompleteView from './AutoCompleteView';
-import TopicAutocomplete from './TopicAutocomplete';
```

Comment:
https://en.wikipedia.org/wiki/Autocomplete

Autocomplete is one word, thus the casing is correct!

Reply:
Done! refactored all to `autocomplete`.
Not sure then why `eslint` is not considering it.